src/Handler/CurlMultiHandler.php

```diff
@@ -32,9 +32,9 @@ class CurlMultiHandler
     private $selectTimeout;
 
     /**
-     * @var resource|\CurlMultiHandle|null the currently executing resource in `curl_multi_exec`.
+     * @var int the currently executing resource in `curl_multi_exec`.
      */
-    private $active;
+    private $active = 0;
```

Comment:
Technically not needed to be initialized as the code checks will still evaluate `null` to falsy, but could potentially be an issue later on if typed properties is added when the minimum version of PHP is upped to at least 7.4